doc/man/hirte-agent.conf.5.md

```diff
@@ -31,8 +31,7 @@ options. The option doesn't have a default value.
 
 #### **ManagerHost** (string)
 
-The host used by `hirte-agent` to connect to `hirte`. Must be a valid IPv4 or IPv6. The option doesn't have a default
-value, it's mandatory to set this option for each hirte-agent.
+The host used by `hirte-agent` to connect to `hirte`. Must be a valid IPv4 or IPv6. The option defaults to 127.0.0.1, it's mandatory to set this option on remote hirte-agents.
```

Comment:
Could you please update the default value also in below files?

https://github.com/containers/hirte/blob/main/config/agent/agent.conf#L14
https://github.com/containers/hirte/blob/main/config/agent/hirte-default.conf#L10
https://github.com/containers/hirte/blob/main/doc/man/hirte-agent.1.md?plain=1#L27